modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/SynonymGraphTokenFilterFactory.java

```diff
@@ -69,6 +69,11 @@ public TokenStream create(TokenStream tokenStream) {
             public AnalysisMode getAnalysisMode() {
                 return analysisMode;
             }
+
+            @Override
+            public String getResourceName() {
```

Comment:
Thanks for the feedback, Carlos. Since this overrides method of a general `TokenFilterFactory` that go beyond `synonyms`,  it is better to keep a general name of `resource`.